src/js/tech/tech.js

```diff
@@ -813,6 +813,8 @@ Tech.withSourceHandlers = function(_Tech){
     // than clear all of our current tracks
     if (this.currentSource_) {
       this.clearTracks(['audio', 'video']);
```

Comment:
@BrandonOCasey is there a reason why we don't want to just *always* call `clearTracks()` here?